### PR TITLE
Np 48098 Add SearchSerialPublicationByQueryHandler in template and openapi spec

### PR DESCRIPTION
**Pull request by monaullah**
- Add `SearchSerialPublicationByQueryHandler` in template and openapi spec

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -419,7 +419,7 @@ paths:
         type: "AWS_PROXY"
       tags:
         - Series
-      summary: Search for series by pid, name, issn, or year
```
Comment (monaullah) [inline docs/openapi.yaml:422]: Support for pid search was removed some time ago. Removing from summary

File: docs/openapi.yaml [hunk 2]
```diff
@@ -576,6 +576,70 @@ paths:
...
+      tags:
+        - SerialPublication
+      summary: Search for series/journal by name, issn, or year
+      description: Returns a list of Series and/or Journals
+      operationId: SearchSerialPublicationByQuery
+      parameters:
+        - in: query
+          name: year
+          schema:
+            type: integer
```
Comment (brinxmat) [inline docs/openapi.yaml:594]: There are more restrictions, no?

Comment (monaullah) [reply docs/openapi.yaml:594]: Not after https://github.com/BIBSYSDEV/nva-publication-channels-java/pull/61

File: docs/openapi.yaml [hunk 3]
```diff
@@ -576,6 +576,70 @@ paths:
...
+          description: The year you want the NVI for
+          required: false
+        - in: query
+          name: query
+          schema:
+            type: string
+          description: The query string, either a free-text, pid or issn
```
Comment (LarsV123) [inline docs/openapi.yaml:601]: Should `pid` be here?

**Review by LarsV123 — APPROVED** [review]

**Review by truhacevkir — APPROVED** [review]

File: docs/openapi.yaml (hunk 3 above)
Comment (monaullah) [reply docs/openapi.yaml]: No, removed in new commit 👍

**Pull request merged**